scripts/bc-wp-init.sh

```diff
@@ -41,19 +41,28 @@ nextip(){
     echo "$NEXT_IP"
 }
 
-# 
-if [[ -n `cat /etc/hosts | grep $domain` ]]
+# WIndows and Linux have diff paths to hosts file
```

Comment:
What about Mac?

Reply:
Add support for MacOs here. Thank you!

Reply:
Please fix typo capital `I` in `WIndows` and please do not use abbreviations like `diff` in comments

Reply:
Remove comments